source/extensions/filters/common/proxy_protocol/BUILD

```diff
@@ -0,0 +1,15 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
```

Comment:
What do you think of a common test that regression tested that thse new proxy protocol outputs can be read by our proxy protocol parser in listener/prot

Reply:
Sounds like a good idea!